lib/apiservers/service/swagger.json

```diff
@@ -900,6 +937,15 @@
       "description": "A VCH host certificate",
       "schema": { "$ref": "#/definitions/PEM" }
     },
+    "vch-log": {
+      "description": "Log messages for a VCH",
+      "schema": {
+        "type": "string"
+      },
+      "examples": {
+        "text/plain": "Sep  8 2017 09:41:56.772-05:00 INFO  ### Installing VCH ####"
```

Comment:
A real example would be really long, so I think I'd just leave the `examples` section off of `vch-log`.